kong/plugins/hmac-auth/access.lua

```diff
@@ -129,6 +149,22 @@ local function validate_clock_skew(headers, date_header_name, allowed_clock_skew
   return true
 end
 
+local function validate_request_body(headers, body)
```

Comment:
@vaibhavatul47 i think this is a good approach, as it could allow us to support additional algorithms in the future.